Assets/HoloToolkit/Utilities/Scripts/Extensions/Extensions.cs

```diff
@@ -1,6 +1,10 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System;
```

Comment:
None of these changes are necessary for input to work, but we found them useful while developing our app.  I'm fine keeping them or removing them.  Anyone feel strongly either way?